docs/reference/configuration.md

```diff
@@ -770,6 +770,9 @@ upstream_oauth2:
         # This helps end user identify what account they are using
         account_name:
           #template: "@{{ user.preferred_username }}"
+      # set to true to allow a user logging in via OIDC to match a pre-existing account instead of failing. 
```

Comment:
```suggestion
      # set to true to allow a user logging in via OIDC to match a pre-existing account instead of failing. 
      # Account matching is based on the localpart template which needs to be set to require or force.
```